WalletWasabi.Fluent/ViewModels/CoinControl/Core/ConstantTemplate.cs

```diff
@@ -0,0 +1,32 @@
+using Avalonia.Controls;
+using Avalonia.Controls.Presenters;
+using Avalonia.Controls.Templates;
+using Avalonia.Layout;
+
+namespace WalletWasabi.Fluent.ViewModels.CoinControl.Core;
+
+public class ConstantTemplate<T> : IDataTemplate
```

Comment:
Why not to use PrivacyTextColumn ?

Reply:
Nobody said we needed to care about privacy mode here.